dm/master/config.go

```diff
@@ -111,6 +111,7 @@ type Config struct {
 
 	MasterAddr    string `toml:"master-addr" json:"master-addr"`
 	AdvertiseAddr string `toml:"advertise-addr" json:"advertise-addr"`
+	OpenAPIAddr   string `toml:"openapi-addr" json:"openapi-addr"`
```

Comment:
Can we just share the same address with `advertise-addr`

Reply:
yes  and I think it's a really good idea! fixed in https://github.com/pingcap/dm/pull/1999/commits/4536f64657cbbadbbf9bd13edcc2e521edc1bacf